inc/ses_to_cloudwatch/namespace.php

```diff
@@ -43,3 +53,21 @@ function on_error_sending_message( Exception $error, $message ) {
 		'message' => $message,
 	] ) );
 }
+
+/**
+ * Truncate string to given maximum size.
+ *
+ * @param string $message     String message to truncate.
+ * @param int    $max_size    Maximum size in bytes, default 5KB.
+ * @param string $replacement String replacement.
+ *
+ * @return string
+ */
+function truncate_message( string $message, int $max_size = 5 * 1024, string $replacement = '…' ) : string {
```

Comment:
This seems maybe `truncate_string` is more appropriate, as from the perspective of this function, it doesn't need to be a "message"